index.js

```diff
@@ -26,6 +26,7 @@ const searchUrl = `${mapzenUrl}/search`
  *   {@link https://mapzen.com/documentation/search/autocomplete/#layers|layer types}
  * @param  {string} [$0.sources='gn,oa,osm,wof']
  * @param  {string} $0.text                       query text
+ * @param {string} [$0.url]                       optional URL to override Mapzen autocomplete endpoint
```

Comment:
Since there is a default value for this parameter, please make a notation of it.